scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -969,7 +998,9 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
       if (advance) next()
       res
     case token: Unquote =>
-      unquote[T]
+      unquote[T](advance)
```

Comment:
Yeah, sure the suggested change would have to be made to both branches. Then the `unquote` method would go back to its old form - without the parameter and without multiple overloads.